### DISCUSSION TITLE
How to exclude DataContext Class generation

### DISCUSSION THREAD

**Discussion by croban** [linq2db/linq2db#3841] [Q&A] [answered]
Hi, how can I exclude DataContext Class generation with CLI?
Somehow ordering of the GetTable<> Properties within, are from computer-to-computer in different order and because of that we have all the time conflicts by pulling git changes.

**Comment by MaceWindu** [comment, marked as answer]
1. ordering fix will be in next release (#3777). 
2. currently there is no option to disable context generation. If you still need it - please create request in issues

**Reply by croban** [reply]
@MaceWindu thx 
for 2) i will solve it over .gitignore and remove it from project file Compile remove ;)
